Jasmine/Levels/ViewModels/LevelSelectorViewModel.swift

```diff
@@ -97,10 +97,10 @@ class LevelSelectorViewModel: LevelSelectorViewModelProtocol {
         case (.swapping, .ciHui):
             return CiHuiSwappingViewModel(
                 time: GameConstants.Swapping.time, gameData: gameData, numberOfPhrases: GameConstants.Swapping.rows)
-        case (.tetris, .chengYu):
```

Comment:
If the game is not implemented it should fatal error, no? Making it fallthrough doesn't really make sense isn't it